src/index.html

```diff
@@ -1,14 +1,53 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet"> -->
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="link-logo" href="#">
+        <img src="./images/logo.png" alt="MOYO">
+      </a>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a class="link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__li">
+            <a class="link" href="#">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Samsung</a>
          </li>
```
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/122379c9-0381-4968-ae0c-eb081c83cd43)

- `nav__item` better describes what it's actually. And as it's a `link` related to `nav` it's better to call it `nav__link`.